include/seastar/core/smp_options.hh

```diff
@@ -21,8 +21,9 @@
 
 #pragma once
 
-#ifndef SEASTAR_MODULE
+#include <seastar/core/resource.hh>
 #include <seastar/util/program-options.hh>
+#ifndef SEASTAR_MODULE
```

Comment:
Why did the #ifndef location change? Typically its scope contains all seastar includes. Not that I understand it.

Reply:
When it compiles src/seastar.cc it includes app-template.hh (line 168) which includes smp_options.hh (this file, line 29) which then needs declarations of resource::cpuset and seastar::program_options from those two headers, which came out of SEASTAR_MODULE scope in app-template.hh before